ConsoleApp1/Program.cs

```diff
@@ -1,9 +1,244 @@
-﻿namespace ConsoleApp1
+﻿using System;
+
+namespace ConsoleApp1
 {
 	internal class Program
 	{
-		static void Main(string[] args)
+        /// <summary>
+        /// SelectionSort algorithm in direct order
+        /// </summary>
+        /// <param name="array">sort this array in direct order</param>
+        static void SelectionSortAsc(int[] array)
```

Comment:
SelectionSortAsc and SelectionSortDesc have only one character diff.  Try to combine it methods, and pass sort order to that methods, do it for the res of the sort methods

Reply:
I united ASC and DESC sort orders in one method, now it looks much better as for me and it allowed siginificantly reduce the code.